fhir-server/src/main/java/com/ibm/fhir/server/resources/FHIRResource.java

```diff
@@ -162,6 +172,17 @@
     private static final String LOCAL_REF_PREFIX = "urn:";
     private static final String HEADERNAME_IF_NONE_EXIST = "If-None-Exist";
     private static final String HEADERNAME_PREFER = "Prefer";
+    private static final String HEADERNAME_IF_MODIFIED_SINCE = "If-Modified-Since";
+    private static final String HEADERNAME_IF_NONE_MATCH = "If-None-Match";
+
+    public static final DateTimeFormatter PARSER_FORMATTER = new DateTimeFormatterBuilder()
```

Comment:
Maybe add a comment indicating what it supports? and format?

Reply:
good point, will add comment

Reply:
done